### PR TITLE
tests: check snapshot in all nodes 

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@BusyJay @hhkbp2 PTAL

### CODE REVIEW

File: tests/raftstore/test_stats.rs [hunk 1]
```diff
@@ -133,13 +133,15 @@ fn test_server_store_snap_stats() {
     must_not_detect_snap(&pd_client);
 }
 
-fn must_detect_snap(pd_client: &Arc<TestPdClient>) {
+fn must_detect_snap(pd_client: &Arc<TestPdClient>, total_nodes: u64) {
```
Comment (siddontang) [inline tests/raftstore/test_stats.rs]: why this test depend on the total nodes?

**Comment by siddontang** [comment]
PTAL @hhkbp2

File: tests/raftstore/test_stats.rs [hunk 2]
```diff
@@ -133,13 +133,15 @@ fn test_server_store_snap_stats() {
...
     for _ in 0..200 {
         sleep_ms(10);
 
-        if let Some(stats) = pd_client.get_store_stats(1) {
-            if stats.get_sending_snap_count() > 0 || stats.get_receiving_snap_count() > 0 {
-                return;
+        for id in 1..total_nodes + 1 {
+            if let Some(stats) = pd_client.get_store_stats(id) {
```
Comment (siddontang) [inline tests/raftstore/test_stats.rs]: I think here we should check whether store 1 has the sending snapshot or store 2 has the receiving snapshot.

**Comment by BusyJay** [comment]
LGTM

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**